### PR TITLE
Setup database connection

### PR DESCRIPTION
**Pull request by Animeshz**

### CODE REVIEW

**Comment by Animeshz** [comment]
A force pull will be required (no need to resolve conflicts), changes are rebased over the tip of main so no need to worry, just that [Removing login page and signup pages](https://github.com/aadi58002/movies-review/commit/e9af48d843e941e2cf4b5384d1732ad317c535bc) commit has been dropped.

File: src/components/AboutUs/AboutUs.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import RaushanImage from '../../images/Raushan-kumar-profile-picture.jpeg';
 import AdityaImage from '../../images/Aditya-Yadav-profile-picture.jpg';
 import AnimeshImage from '../../images/Animesh-sahu-profile-picture.jpeg';
 import SatvikImage from '../../images/Satvik-Dubey-Profile-Picture.jpg';
-// import Image from '../../images/';
+import ManiImage from '../../images/ManiRajGupta_profile_picture.jpeg';
```
Comment (ManiRajGupta) [inline src/components/AboutUs/AboutUs.js]: photo uploaded

**Review by ManiRajGupta — APPROVED** [review]

**Comment by aadi58002** [comment]
Adding database connectivity and changing the website working acc to the database

File: src/components/AboutUs/AboutUs.js (hunk 1 above)
Comment (Animeshz) [reply src/components/AboutUs/AboutUs.js]: ~~Its already there https://github.com/Animeshz/movies-review/blob/main/src/components/AboutUs/AboutUs.js#L15 in my branch, its probably shown due to the commit is not dropped.~~

Wait this is just diff, it means its been added, what is this comment about :thinking: I thought you were trying to suggest something.

**Pull request merged**